packages/checked_oint/checked_oint.0.1.0/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "An OCaml library for checked integer arithmetic"
+maintainer: ["Tima Kinsart <hirrolot@gmail.com>"]
+authors: ["Tima Kinsart <hirrolot@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/Hirrolot/checked_oint"
+bug-reports: "https://github.com/Hirrolot/checked_oint/issues"
+depends: [
+  "ocaml"
```

Comment:
```suggestion
  "ocaml" {>= "4.13"}
```
Due to
```
# (cd _build/default && /home/opam/.opam/4.12/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.checked_oint.objs/byte -I /home/opam/.opam/4.12/lib/bigarray-compat -I /home/opam/.opam/4.12/lib/ctypes -I /home/opam/.opam/4.12/lib/ctypes-foreign -I /home/opam/.opam/4.12/lib/integers -I /home/opam/.opam/4.12/lib/ocaml/threads -I /home/opam/.opam/4.12/lib/ppx_deriving/runtime -I /home/opam/.opam/4.12/lib/ppx_enumerate/runtime-lib -I /home/opam/.opam/4.12/lib/result -I /home/opam/.opam/4.12/lib/stdlib-shims -intf-suffix .ml -no-alias-deps -o lib/.checked_oint.objs/byte/checked_oint.cmo -c -impl lib/Checked_oint.pp.ml)
# File "lib/Checked_oint.ml", line 289, characters 11-29:
# 289 |         if String.starts_with ~prefix s
#                  ^^^^^^^^^^^^^^^^^^
# Error: Unbound value String.starts_with
```
and similar other errors using operators or functions not yet available in older compilers